.claude/commands/savetmp.py

```
@@ -123,52 +129,87 @@ def _resolve_repo_info(
             )
+
+            # Log the result of the two-dot diff attempt
+            if two_dot_output is None:
+                if changed_files_output is None:
+                    logging_util.warning(
+                        f"Both git diff strategies failed for base={base_ref}. "
+                        f"Possible causes: network issues, git corruption, or invalid ref. "
+                        f"changed_files will be empty."
+                    )
+                else:
+                    logging_util.debug(f"Two-dot fallback failed, keeping 3-dot result (empty).")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)